Resources/Prototypes/Reagents/gases.yml

```diff
@@ -254,7 +254,8 @@
           shouldHave: false
         key: ForcedSleep
         component: ForcedSleeping
-        time: 3
+        time: 250 # time in miliseconds (due to how respiration works this needs to be over 2 seconds to prevent losing the effect.)
+        refresh: true
```

Comment:
You can't just say it's milliseconds and have it work, this is deserialized as seconds.